torch/csrc/jit/runtime/profiling_graph_executor_impl.h

```diff
@@ -14,6 +14,16 @@ struct ProfilingGraphExecutorImpl : public GraphExecutorImplBase {
   GraphExecutorState getDebugState() override;
   ~ProfilingGraphExecutorImpl() override = default;
 
+  void flushCompilationCache() {
```

Comment:
you might want to add some underscores here, or call it `debugFlushCompilationCache`. We dont want anyone else to use this really..

Reply:
I'll rename it to `debug`